src/w2popup.js

```diff
@@ -989,3 +989,95 @@ var w2confirm = function (msg, title, callBack) {
         }
     };
 };
+
+var w2prompt = function (options, callBack) {
```

Comment:
I think to make it similar with w2alert and w2confirm, the arguments should be
```js
w2prompt = function (label, title, callBack)
```
Then, if label is an object - assume it is extended options for the prompt